sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -877,7 +877,7 @@
       "get": {
         "summary": "Gets a module twin.",
         "description": "Gets the module twin. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-device-twins for more information.",
-        "operationId": "Twin_GetModuleTwin",
+        "operationId": "Modules_GetTwin",
```

Comment:
>Modules [](start = 24, length = 7)

I think separating the modules and devices in a larger group "twins" is not a good idea. It leads to pollution of this APIs. The "Twin" or "Twins" group was c;loser to the reality. Doesn't matter if it is a module or device. Twin group is a higher separation than device and module. If tomorrow we come up with another location of a device we will create a third group for XXX_GetTwins? No.... (Module is just a device in a different context...)

Reply:
Discussed offline, we will be going with a proposal of operation id grouping for Devices and Modules